cap_client/cap_api.py

```diff
@@ -154,21 +154,22 @@ def get_shared(self, pid=None, all=False):
 
         return response if pid else response['hits']['hits']
 
-    def get_schema(self, ana_type=None, version='0.0.1'):
+    def get_schema(self, ana_type=None, record=False):
```

Comment:
like in an [issue description](https://github.com/cernanalysispreservation/cap-client/issues/107) - remove the **default value** for `version` parameter (but not parameter itself). So if version is not specified you don't pass it to an endpoint - it will handle it on its own, by returning the latest version of a given schema (which makes more sense than fetching version 0.0.1 by default ;) )